frontend/test/metabase/scenarios/question/nulls.cy.spec.js

```diff
@@ -192,7 +192,7 @@ describe("scenarios > question > null", () => {
       summarize();
       // remove pre-selected "Count"
       cy.icon("close").click();
-      // dropdown immediately opens with the new set of metrics to choose from
+      cy.findByText("Add a metric").click();
```

Comment:
It was actually working kinda weird on `master`. If you've opened an ad-hoc question, opened a summarize sidebar, and tried to remove the default "count" aggregation, it wouldn't actually remove it, but just open a popover for adding a new aggregation. After the changes in this PR, it would actually remove the "count" aggregation and you'd need to click "Add a metric" to add a new one

Reply:
This is awesome!